rasterimage/renderer.go

```diff
@@ -1,32 +1,41 @@
-package canvas
+package rasterimage
```

Comment:
A nitpick, but I think I'd prefer `rasterizer`. People would write `rasterizer.New(img, dpm)`, i.e. instantiate a new rasterizer that writes to an image `img`. Additionally, `rasterizer.PNGWriter` would be a PNG writing rasterizer.